src/bigint.rs

```diff
@@ -366,6 +366,16 @@ impl Num for BigUint {
     /// Creates and initializes a `BigUint`.
     fn from_str_radix(s: &str, radix: u32) -> Result<BigUint, ParseBigIntError> {
         assert!(2 <= radix && radix <= 36, "The radix must be within 2...36");
+        let mut s = s;
+        if s.starts_with('+') {
+            s = &s[1..];
+            if s.starts_with('+') {
+                // create ParseIntError::InvalidDigit
```

Comment:
I did create a test for it, see line 3767. The reason I need to check for this separately is because the error path in the byte-conversion step is `Ok` with `"+21"`.